spec/shr_genetics.txt

```diff
@@ -19,7 +19,7 @@ Concept:		MTH#C0017393  // should be a finding (Q+A), not an observable
 Description:	"A finding with respect to variation in a segment of DNA with an identifiable physical location on a chromosome."
 				FindingTopicCode is MTH#C0017393 "A segment of DNA with an identifiable physical location on a chromosome and whose inheritance can be followed. (source: NCI Metathesaurus)"
 0..0			FindingResult // panel, no numerical or coded result
-				SpecificFocusOfFinding value is type ref(CancerDisorderPresent) // maybe. More likely just the specimen 
+				SpecificFocusOfFinding value is type ref(Specimen) // maybe. More likely just the specimen 
 //  What gene are we looking at?
```

Comment:
@markkramerus - This is specified under the element GeneName.

Reply:
Specimen is always part of LaboratoryObservation. I should have said NO SpecificFocusOfFinding is necessary because we already have Specimen.  The right thing to do, I think, is 0..0 SFoF.

Reply:
@markkramerus - done. zero'd out SpecificFocusOfFinding.